### PR TITLE
COMMERCE-12716 update payment status only if it is passed

### PR DESCRIPTION
**Pull request by jaydawu**
https://liferay.atlassian.net/browse/COMMERCE-12716

Resent from https://github.com/liferay-commerce/liferay-portal/pull/4409

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONOrdersAPI.macro [hunk 1]
```diff
@@ -661,6 +661,36 @@ definition {
 		return ${response};
 	}
 
+	@summary = "Default summary"
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONOrdersAPI.macro:664]: Can you update the summary part and put a simple description of what the macro is doing?

Something like: "This macro can be used to update order via API" etc.

**Review by baharturk — CHANGES_REQUESTED** [review]
Please take a look at the comment I left. Thanks!

The test is all good, just a simple request for the summary part.

**Review by baharturk — APPROVED** [review]

**Comment by jaydawu** [comment]
ci:test:sf

**Review by baharturk — APPROVED** [review]

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
No relevant errors to this pull, it can be force-forwarded, thanks!

cc/ @victoroscuro

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**